samples/tacos/build.gradle.kts

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2023 Slack Technologies, LLC
+// SPDX-License-Identifier: Apache-2.0
+import org.jetbrains.kotlin.gradle.internal.KaptGenerateStubsTask
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+  id("com.android.application")
+  kotlin("android")
+  kotlin("kapt")
+  kotlin("plugin.parcelize")
+  alias(libs.plugins.ksp)
+}
+
+android {
+  namespace = "com.slack.circuit.tacos"
+
+  defaultConfig {
+    minSdk = 33
+    testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
```

Comment:
I think both of these are handled automatically by our setup in the root build file

Reply:
Looks like I can remove `testInstrumentationRunner`. Removing `minSdk` results in an error:
```
In order to use core library desugaring, please enable multidex.
```
We appear to set a default `minSdk` for library and test but not application